spec/components/one_time_code_input_component_spec.rb

```diff
@@ -58,6 +58,30 @@
     end
   end
 
+  describe 'hint' do
+    it 'includes hint text as a descriptor of the field' do
+      descriptors = rendered.
+        at_css('.one-time-code-input__input')['aria-describedby'].
+        split(' ').
+        map { |descriptor_id| rendered.at_css("##{descriptor_id}")&.text }
```

Comment:
This pattern could be a useful generic test helper, i.e. get the texts describing a field via `aria-describedby` relationship.

Reply:
Maybe something like...

```rb
expect(field).to have_description(...)
```

Reply:
Extracted new `have_description` matcher in d95511906.